proof/access-control/RISCV64/ArchSyscall_AC.thy

```diff
@@ -168,7 +169,7 @@ global_interpretation Syscall_AC_1?: Syscall_AC_1
 proof goal_cases
   interpret Arch .
   case 1 show ?case
-    by (unfold_locales; (fact Syscall_AC_assms | wp init_arch_objects_inv))
+    by (unfold_locales; wpsimp wp: Syscall_AC_assms)
```

Comment:
same as my other comment, is it only wp rules in the assms? surprised it doesn't work with `fact`

Reply:
See my response in the other thread